core/src/test/java/com/google/errorprone/bugpatterns/testdata/AbortInOvercatchPositiveCases.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright 2011 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.google.errorprone.bugpatterns;
+
+/**
+ * @author yuan@ece.toronto.edu (Ding Yuan)
+ */
+public class AbortInOvercatchPositiveCases {
+  public void error() throws IllegalArgumentException {
+	throw new IllegalArgumentException("Fake exception.");
+  }
+  
+  public void abortInThrowable() {
+    int a = 0;
+    try {
+      error();
+    } // BUG: Diagnostic contains: 
+    catch (Throwable t) {
+      System.exit(1);
+    }
+  }
+  
+  public void abortInException() {
+    int a = 0;
+    try {
+      error();
+    } // BUG: Diagnostic contains: 
+    catch (Exception e) {
+      System.exit(1);
+    }
+  }
+}
```

Comment:
Can you please add test cases for abort methods other than `System.exit()`?  Please add at least one each for methods named `abort` and `shutdown`.